index.js

```diff
@@ -91,33 +91,56 @@ const stonksifyRules = rules.sort(function compare(a, b) {
   }
 });
 
-input.on("keyup", function (e) {
+input.on("keyup", function(e) {
   if (e.which === 13) {
     console.log("Hit Enter");
     enterWord(input.val());
   }
 });
 
-submitButton.on("click", function () {
+submitButton.on("click", function() {
   console.log("Clicked Submit");
   enterWord(input.val());
 });
 
-shareButton.on("click", function () {
+shareButton.on("click", function() {
   copyToClipboard();
-  shareButton.text("Copied to Clipboard!");
+  // adds a span to display "Link Copied"
+  // then, styles the text above the input field's position and animates the text to float upward before removing from the dom
+  input.after('<span id="copied_popup">Link Copied!</span>');
+  let copiedTop = input.offset().top - input.height() - 5;
+  let copiedLeft = input.offset().left + 40;
+  $("#copied_popup")
+    .css("position", "absolute")
```

Comment:
Let's try and keep most CSS styles to style.css and just reference the class name in here.

Reply:
good call here. moved